.github/workflows/build-builder.yaml

```diff
@@ -0,0 +1,35 @@
+name: build-builder
+
+on:
+  push:
+    branches:
+    - main
+
+jobs:
+  build-and-push-image:
+    runs-on: ubuntu-22.04
```

Comment:
should be 24.04  ?

Reply:
oh interesting I trust on PD repo so I pick the version from there :joy: let me check and update from official gh doc

Reply:
I think we're in the process of updating it

Reply:
done